odoo/addons/test_new_api/tests/test_new_fields.py

```diff
@@ -2457,8 +2477,97 @@ class TestSelectionDeleteUpdate(common.TransactionCase):
     MODEL_ABSTRACT = 'test_new_api.state_mixin'
 
     def test_unlink_asbtract(self):
-        self.env['ir.model.fields.selection'].search([
+        selections = self.env['ir.model.fields.selection'].search([
             ('field_id.model', '=', self.MODEL_ABSTRACT),
             ('field_id.name', '=', 'state'),
             ('value', '=', 'confirmed'),
-        ], limit=1).unlink()
+        ], limit=1)
+        if selections.pool.ready:
+            # Ensures the test works fine in post-install
+            # for local tests
+            self.env.cr.execute("""
+                WITH fields AS (
+                    Select field.id
+                    from ir_model_fields field
+                    JOIN ir_model_fields_selection selection on field_id=field.id
+                    WHERE selection.id=%i
+                )
+                UPDATE ir_model_fields field
+                SET state = 'manual'
+                FROM fields f
+                WHERE f.id=field.id
+            """% selections.id)
+            selections.invalidate_cache()
+        selections.unlink()
+
+
+@common.tagged('prepostcomputes')
+class TestPrePostComputes(common.TransactionCase):
+
+    def test_pre_post_create_computes(self):
+        Model = self.env["test_new_api.model_advanced_computes"]
+
+        # Ensure automatic assignation of pre_compute=False works
+        # 1) dependency on create_date/create_uid/write_date/write_uid
+        self.assertFalse(Model._fields.get('create_month').pre_compute)
+        # 2) dependency on a pre_compute=False field
+        self.assertFalse(Model._fields.get('full_card_content').pre_compute)
```

Comment:
To remove, since optin feature, or force pre_compute=True on the fields then check they are still put to pre_compute=False ?